app/controllers/register.js

```diff
@@ -22,11 +22,8 @@ export default class RegisterController extends Controller {
     this.model.save()
       .then(user => {
         this.set('session.newUser', user.get('email'));
-        if (this.inviteToken) {
-          this.send('loginExistingUser', user.get('email'), password, this.inviteToken, this.event);
-        } else {
-          this.transitionToRoute('login');
-        }
+        this.send('loginExistingUser', user.get('email'), password, this.inviteToken, this.event);
+        this.transitionToRoute('/');
```

Comment:
Why is this needed? Shouldn't the above action be awaited?

Reply:
> Shouldn't the above action be awaited?

why? I am not getting it.